### PR TITLE
icon coloring and opacity in SLD

### PR DESCRIPTION
**Pull request by ischneider**
see #132 

this still needs tests

### CODE REVIEW

**Comment by bartvde** [comment]
I'll merge and add tests with a separate PR.

File: lib/edit/style/SLDStyleConverter.js [hunk 1]
```diff
@@ -98,6 +101,10 @@ exports.SLDStyleConverter = function() {
                         wellKnownName: style.symbol && style.symbol.shape || 'circle'
                     }];
             }
+            var opacity = 1;
+            if (style.symbol && angular.isDefined(style.symbol.fillOpacity)) {
+                opacity = Math.max(0.01, style.symbol.fillOpacity) / 100;
```
Comment (bartvde) [inline lib/edit/style/SLDStyleConverter.js]: hey @ischneider one thing I am wondering here (but I don't know the SLD spec specifics nor the GeoServer internals), will this not lead to the fill opacity being "executed" twice? i.e. 80*80% = 64% ?

**Comment by bartvde** [comment]
hey @ischneider wondering why this is not working on my local instance here

```
<sld:StyledLayerDescriptor xmlns:sld="http://www.opengis.net/sld" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:ogc="http://www.opengis.net/ogc" version="1.0.0"><sld:NamedLayer><sld:Name>tasmania_cities</sld:Name><sld:UserStyle><sld:FeatureTypeStyle><sld:Rule><sld:PointSymbolizer><sld:Graphic><sld:ExternalGraphic><sld:OnlineResource xlink:href="http://104.236.102.193/icons/icon/5526?fill=%23FF0000&amp;stroke=%23000000"/><sld:Format>image/svg+xml</sld:Format></sld:ExternalGraphic><sld:Opacity>1</sld:Opacity><sld:Size>20</sld:Size></sld:Graphic></sld:PointSymbolizer></sld:Rule></sld:FeatureTypeStyle></sld:UserStyle></sld:NamedLayer></sld:StyledLayerDescriptor>
```

screen shot 2015-02-02 at 12 29 23

**Comment by bartvde** [comment]
Ah on another icon it works though, weird

```
<sld:StyledLayerDescriptor xmlns:sld="http://www.opengis.net/sld" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:ogc="http://www.opengis.net/ogc" version="1.0.0"><sld:NamedLayer><sld:Name>tasmania_cities</sld:Name><sld:UserStyle><sld:FeatureTypeStyle><sld:Rule><sld:PointSymbolizer><sld:Graphic><sld:ExternalGraphic><sld:OnlineResource xlink:href="http://104.236.102.193/icons/icon/2906?fill=%23FF0000&amp;stroke=%23000000"/><sld:Format>image/svg+xml</sld:Format></sld:ExternalGraphic><sld:Opacity>1</sld:Opacity><sld:Size>20</sld:Size></sld:Graphic></sld:PointSymbolizer></sld:Rule></sld:FeatureTypeStyle></sld:UserStyle></sld:NamedLayer></sld:StyledLayerDescriptor>
```

screen shot 2015-02-02 at 12 30 31

File: lib/edit/style/SLDStyleConverter.js (hunk 1 above)
Comment (ischneider) [inline lib/edit/style/SLDStyleConverter.js]: @bartvde I guess it's worth looking at but I found opacity wasn't working until this was added

**Pull request merged**